user-manuals/administration/facilities/manage-facility-staff.md

```diff
@@ -1,23 +1,27 @@
-# Manage facility Staff
+# Manage Facility Staff
 
-To effectively oversee a facility's staff, which includes roles such as warehouse picker, warehouse packer, warehouse clerk, warehouse receiver, and warehouse manager, the store manager assigns these roles to staff members. Subsequently, staff members can access the HotWax Commerce app and perform functions specific to their assigned roles. Users can follow these steps to efficiently allocate role-specific permissions for accessing HotWax Commerce and its associated applications.
+To effectively oversee a facility's staff, which includes roles such as warehouse picker, warehouse packer, warehouse clerk, warehouse receiver, and warehouse manager, the store manager assigns these roles to staff members. Subsequently, staff members can access the HotWax Commerce apps and perform functions specific to their assigned roles. Users can follow these steps to efficiently allocate role-specific permissions to staff members for accessing HotWax Commerce and its associated applications.    
 
-**Add Staff**
+ &nbsp;
```

Comment:
Can you describe what this is meant to do? @sanjanashrivastava

Reply:
Initially it looked a little closely spaced to me, so I was checking how it would look with line breaks. Edited it, please check now.

Reply:
Understood, you can add line breaks by just adding extra spaces as well FYI